security/passage/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        FiloSottile passage 1.7.4a1
+categories          security
+license             GPL-2+
+platforms           any
+supported_archs     noarch
+
+name                passage
+version             1.7.4a1
+revision            3
+maintainers         {macports.halostatue.ca:austin @halostatue} \
+                      openmaintainer
+description         ${name} is the standard unix password manager
+long_description    {*}${description}
+homepage            https://www.passwordstore.org/
+
+# passage depends on either `age` or `rage`
+depends_run         port:git \
+                    port:tree \
+                    port:qrencode \
+                    port:util-linux \
+                    bin:age:age
+
+checksums           rmd160  257e470019e789736d7eaf203c5eb9a153ea1525 \
+                    sha256  3b63fb2cf373cca2409b170867367cb1503f3d91f25d7c50d7d4d85efdc3b02f \
+                    size    19601
+
+patchfiles          patch-getopt-path.diff
+
+post-patch {
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/src/platform/darwin.sh
+}
+
+use_configure       no
+build {}
+
+destroot.env-append PREFIX=${prefix} \
+                    SYSCONFDIR=${prefix}/etc \
+                    WITH_ALLCOMP=yes
+
+# Add misc contrib files (not currently in passage repo)
+# post-destroot {
+#     # set vim-plugin ${prefix}/share/vim/vimfiles
+#     # xinstall -d ${destroot}${vim-plugin}/ftdetect
+#     # xinstall -m 0644 ${worksrcpath}/contrib/vim/redact_pass.vim \
+#     #     ${destroot}${vim-plugin}/ftdetect
+
+#     set emacs-plugin ${prefix}/share/emacs/site-lisp
+#     xinstall -d ${destroot}${emacs-plugin}
+#     xinstall -m 0644 ${worksrcpath}/contrib/emacs/password-store.el \
+#         ${destroot}${emacs-plugin}
+
+#     xinstall -d ${destroot}${prefix}/share/${name}
+#     xinstall -m 0644 ${worksrcpath}/contrib/pass.applescript \
+#         ${destroot}${prefix}/share/${name}
+# }
```

Comment:
don't add commented out lines

Reply:
I put my reasoning on why I left these [in an earlier comment](https://github.com/macports/macports-ports/pull/21117#issuecomment-1783715609); I can remove them, but what I *actually* want to do is to figure out how they could be included. As noted, `passage` is frontend compatible with `pass` (in the same way that `gopass` is compatible with `pass`), so if there is a way to either get them in FiloSottile/passage or to also download the `pass` tarball and apply patches to the `contrib/` directory to be compatible with `passage` and install these as `passage.applescript`, etc. I would prefer it.

This can be deferred to a future update, though.